CHANGELOG.md

```diff
@@ -0,0 +1,318 @@
+# Fastlane CHANGELOG
+
+## Master
+
+* None.  
```

Comment:
so after every release we want to reset the top of the file to have `Master` `None`? Can we remove the `.` here?